utils/defillama.ts

```diff
@@ -7,19 +7,37 @@ interface TvlData {
   totalLiquidityUSD: number;
 }
 
-export async function getDefillamaTvl() {
-  const { data } = await axios.get(baseUrl);
+export async function getDefillamaTvl(url: string = baseUrl) {
+  const { data } = await axios.get(url);
   const defillamaTvlData: TvlData[] = data.tvl.slice(-1);
   const latestTvl = defillamaTvlData[0].totalLiquidityUSD;
   return latestTvl;
 }
 
-export async function getDefillamaPercentChange() {
-  const { data } = await axios.get(baseUrl);
+export async function getDefillamaPercentChange(url: string = baseUrl) {
+  const { data } = await axios.get(url);
   const defillamaTvlData: TvlData[] = data.tvl.slice(-2);
   const latestTvl = defillamaTvlData[1].totalLiquidityUSD;
   const previousDayTvl = defillamaTvlData[0].totalLiquidityUSD;
   const defillamaPercentChange =
     Math.round(((latestTvl - previousDayTvl) / previousDayTvl) * 1000) / 10;
   return defillamaPercentChange;
 }
+
+export async function getDefillamaStats(url: string = baseUrl) {
+  const { data } = await axios.get(url);
+
+  const latestTvlData: TvlData[] = data.tvl.slice(-1);
+  const tvl = latestTvlData[0].totalLiquidityUSD;
+
+  const twoDaysTvlData: TvlData[] = data.tvl.slice(-2);
+  const latestTvl = twoDaysTvlData[1].totalLiquidityUSD;
+  const previousDayTvl = twoDaysTvlData[0].totalLiquidityUSD;
+  const tvl24hChange =
```

Comment:
did not realize we could get this from defi llama. it looks like you are able to get 24 hour changes for uma and each contract individually?

Reply:
I can't do that for each contract.